app/src/main/res/layout/activity_rider_view_request.xml

```diff
@@ -120,6 +120,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginTop="5dp"
+            android:id="@+id/imageView_requestStatus"
```

Comment:
I don't know if this is caused by your code but the status icon for every request is kind of super pushed into the side and difficult to see.